<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMyImageEditorUI</class>
 <widget class="QMainWindow" name="CMyImageEditorUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Image Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="CHistogramViewWrpr" name="m_histogramView">
   <property name="floating">
    <bool>false</bool>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="CDirectCamViewWrpr" name="m_directCamView">
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="CLoggingViewWrpr" name="m_loggingView">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="CImageEditorViewWrpr" name="m_imageEditorView">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDirectCamViewWrpr</class>
   <extends>QDockWidget</extends>
   <header>..\include\dockwidgets\directcamview_wrpr.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CHistogramViewWrpr</class>
   <extends>QDockWidget</extends>
   <header>..\include\dockwidgets\histogramview_wrpr.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CLoggingViewWrpr</class>
   <extends>QDockWidget</extends>
   <header>..\include\dockwidgets\loggingview_wrpr.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CImageEditorViewWrpr</class>
   <extends>QDockWidget</extends>
   <header>..\include\dockwidgets\imageeditorview_wrpr.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
